content/docs/command-reference/metrics/index.md

```diff
@@ -92,6 +92,16 @@ DVC itself does not ascribe any specific meaning for these numbers. Usually they
 are produced by the model training or model evaluation code and serve as a way
 to compare and pick the best performing experiment.
 
+<admon type="tip">
+
+See
+[the guide on metrics badges](/doc/start/data-management/metrics-parameters-plots#badges)
+for prettifying a repository's with an automatically updated badge like this:
+
+[![DVC-metrics](https://img.shields.io/badge/dynamic/json?url=https://github.com/iterative/example-get-started/raw/main/evaluation.json&query=avg_prec&label=Average%20Precision&colorA=grey&colorB=F46737)](https://github.com/iterative/example-get-started/raw/main/evaluation.json)
```

Comment:
```suggestion
[![DVC-metrics](https://img.shields.io/badge/dynamic/json?url=https://github.com/iterative/example-get-started/raw/main/evaluation.json&query=avg_prec&label=Average%20Precision&colorA=grey&colorB=F46737&logo=DVC)](https://github.com/iterative/example-get-started/raw/main/evaluation.json)
```

Reply:
Yep, if we show a single badge, it makes to include logo. I the guide we can put a link on how to customize it

Reply:
The engine adds an "external link" icon to it which doesn't look great

<img width="352" alt="image" src="https://user-images.githubusercontent.com/1477535/172074500-4998b930-aa6c-43d4-bbb7-7f54ca09e90f.png">

Reply:
p.s. I also don't think we need this in the cmd ref as it doesn't have much to do with the `metrics` command usage.

Reply:
```suggestion
<a href="https://github.com/iterative/example-get-started/raw/main/evaluation.json"><img alt="DVC-metrics" src="https://img.shields.io/badge/dynamic/json?url=https://github.com/iterative/example-get-started/raw/main/evaluation.json&query=avg_prec&label=Average%20Precision&colorA=grey&colorB=F46737&logo=DVC"/></a>
```